Library/Formula/icecream.rb

```diff
@@ -0,0 +1,92 @@
+class Icecream < Formula
+  homepage ""
+  url "https://github.com/icecc/icecream/archive/1.1rc1.tar.gz"
+  version "1.1rc1"
+  sha1 "ccde08f67297122710270b440492a528df042f52"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "docbook2X" => :build
+
+  def install
+    system "./autogen.sh"
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+    (prefix+'org.opensuse.icecc.plist').write iceccd_plist
+    (prefix+'org.opensuse.icecc-scheduler.plist').write scheduler_plist
+  end
+
+  def caveats
+    return <<-EOS
+To override the toolset with icecc, add to your path:
+   #{prefix}/bin/icecc-toolset
+
+To setup the icecc daemon:
+    cp #{prefix}/org.opensuse.icecc.plist ~/Library/LaunchAgents/
+    launchctl load -w ~/Library/LaunchAgents/org.opensuse.icecc.plist
+
+To setup the icecc scheduler:
+    cp #{prefix}/org.opensuse.icecc-scheduler.plist ~/Library/LaunchAgents/
+    launchctl load -w ~/Library/LaunchAgents/org.opensuse.icecc-scheduler.plist
+
+If this is an upgrade, to restart a previous setup of the icecc daemon or scheduler:
+    launchctl unload -w ~/Library/LaunchAgents/org.openuses.icecc.plist
+    launchctl load -w ~/Library/LaunchAgents/org.openuses.icecc.plist
+ or
+    launchctl unload -w ~/Library/LaunchAgents/org.openuses.icecc-scheduler.plist
+    launchctl load -w ~/Library/LaunchAgents/org.openuses.icecc-scheduler.plist
+EOS
```

Comment:
We don't need instructions for each of the plists; at least one of them will be printed out by `brew info` already.